src/test/java/org/mitre/synthea/world/geography/LocationTest.java

```diff
@@ -102,11 +102,11 @@ public void testTimezone() {
   @Test
   public void testAllDemographicsHaveLocations() throws Exception {
     String demoFileContents =
-        Utilities.readResource(Config.get("generate.demographics.default_file"));
+        Utilities.readResourceOrPath(Config.get("generate.demographics.default_file"));
     List<LinkedHashMap<String, String>> demographics = SimpleCSV.parse(demoFileContents);
 
     String zipFileContents =
-        Utilities.readResource(Config.get("generate.geography.zipcodes.default_file"));
+        Utilities.readResourceOrPath(Config.get("generate.geography.zipcodes.default_file"));
```

Comment:
Technically, these are both CSV files, so should we use `readResource(filename,  true, true)`?